### DISCUSSION TITLE
Model.Init vs sequelize.define

### DISCUSSION THREAD

**Discussion by smuralimohan** [sequelize/sequelize#13794] [Help & Questions]
Hello Sequelize team,

I am using Sequelize v6("sequelize": "^6.12.0") with Typescript.  When I tried to create models using the following syntax, 
sequelizeInstance.define**<User>**(...)
Typescript complains with the error:  'Untyped function calls may not accept type arguments. ts(2347)" 

I have the following packages installed as dev-dependencies:

"typescript": "^4.5.2",
"sequelize-typescript": "^2.1.1",
 "@types/node": "^17.0.1",
 "@types/sequelize": "^4.28.10",
 "@types/validator": "^13.7.0",

Please let me know if there is anything I am missing. 
In general, wanted to know if sequelize.define() is going to be deprecated going forward. What is the recommended way to create Models? Model.Init() or Sequelize.define()? 

Inputs appreciated.

Thanks.
Murali Mohan

**Comment by ephys** [comment]
If you're using sequelize 6, then remove `@types/sequelize`. sequelize 6 has built in types.

If you're importing sequelize from [sequelize-typescript](https://github.com/RobinBuschmann/sequelize-typescript), note that it is a separate project and issues should be opened there as it's a different team.

I'm afraid I can't help more than that with a reproducible example.

Both `Model.init` & `Sequelize.define` are likely to stay, I would however recommend going for the class-based approach and used `Model.init`
